packages/quick_actions/quick_actions_ios/ios/quick_actions_ios.podspec

```diff
@@ -14,11 +14,14 @@ Downloaded by pub (not CocoaPods).
   s.author           = { 'Flutter Dev Team' => 'flutter-dev@googlegroups.com' }
   s.source           = { :http => 'https://github.com/flutter/plugins/tree/main/packages/quick_actions' }
   s.documentation_url = 'https://pub.dev/packages/quick_actions'
-  s.source_files = 'Classes/**/*.{h,m}'
-  s.public_header_files = 'Classes/*.h'
-  s.private_header_files = 'Classes/PrivateHeaders/*.h'
+  s.swift_version = '5.0'
+  s.source_files = 'Classes/**/*.{h,m,swift}'
+  s.xcconfig = {
+     'LIBRARY_SEARCH_PATHS' => '$(TOOLCHAIN_DIR)/usr/lib/swift/$(PLATFORM_NAME)/ $(SDKROOT)/usr/lib/swift',
+     'LD_RUNPATH_SEARCH_PATHS' => '/usr/lib/swift',
+  }
+  s.public_header_files = 'Classes/**/*.h'
```

Comment:
unfortunately we have to expose private headers until migration is complete (refer to the PR description)